pkgs/applications/networking/cluster/nomad/default.nix

```diff
@@ -62,7 +61,7 @@ rec {
   nomad_1_5 = throwUnsupportaed "nomad_1_5";
 
   nomad_1_6 = generic {
-    buildGoModule = buildGo121Module;
```

Comment:
Would you be able to remove nomad 1.6 entirely instead?

Reply:
Sure!

Reply:
Done.